src/config/AppPaths.ts

```diff
@@ -10,6 +10,8 @@ enum APP_PATHS {
   Rewards = '/points',
   RewardsActivities = '/points/activity',
   RewardsLeaderboard = '/points/leaderboard',
+  PointsVault = '/points/vault',
+  PointsVaultLogin = '/points/vault/login',
```

Comment:
@mishramonalisha76 i don't think we need 2 separate routes

Reply:
okay i was thinking the same but i though this is a requirement, changing it now